cmd/flags.go

```diff
@@ -229,7 +229,7 @@ func retryFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
 }
 
 func updateTimeFlags(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
-	cmd.Flags().Duration(flagThresholdTime, 6*time.Hour, "time before to expiry time to update client")
+	cmd.Flags().Duration(flagThresholdTime, relayer.DefaultClientUpdateThreshold, "time before to expiry time to update client")
```

Comment:
In combination with this suggestion https://github.com/cosmos/relayer/pull/1008/files#r1000932480

```suggestion
	cmd.Flags().Duration(flagThresholdTime, relayer.DefaultClientUpdateThreshold, "time after previous client update before automatic client update")
```